spec/behavior/iomonad.spec.js

```diff
@@ -186,7 +186,7 @@ describe( "IO Monad", function() {
 		var acquiredHandle, acquiringHandle;
 		before( function( done ) {
 			var factory = function() {
-				return when.promise( function( resolve ) {
+				return new Promise( function( resolve ) {
```

Comment:
what's happin here?!

Reply:
https://github.com/cujojs/when/blob/master/docs/api.md#whenpromise

Create a Promise, replaced with new Promise(...

Reply:
i meant that it just has the resolve method.... what about rejection? no need to take care of it here as it's a test case?